tokio/src/time/driver/mod.rs

```diff
@@ -333,28 +334,33 @@ impl Inner {
         self.elapsed.load(SeqCst)
     }
 
+    #[cfg(test)]
+    #[cfg(loom)]
```

Comment:
nit, take it or leave it: i think this is equivalent to
```rust
#[cfg(all(test, loom))]
```
which i think might be a little clearer — it took me a minute to determine if the intention was to include this with `cfg(loom)` _or_ `cfg(test)` or only with `cfg(loom)` _and_ `cfg(test)`.

Reply:
Oh, the style you've posted is much nicer. I'll clean that up.

Reply:
Addressed in 20d01c6.